string message_type
string file_path
---
bool success
